### ISSUE TITLE
[A11Y] Editorial View - Body text is too small

### ISSUE THREAD

**Issue by blesildaramirez** [pkp/pkp-lib#12688]
## Issue Description
Some body text in Editorial View is set to `0.75rem (12px)` across the workflow pages. For comparison, Reader View body is `0.93rem` and is being bumped to `1rem` in https://github.com/pkp/pkp-lib/issues/12685. This affects all areas where the class `text-base-normal` or `text-sm-normal` are used for vue rendered components. Some other legacy forms could have dedicated form field font sizes, so needs to investigate further.

## Target framing:
`12px (0.75rem)` was flagged by the audit as too small. We can reasonably target either `16px (1rem, ideal)` or `14px (0.875rem, a good middle ground)`. Both should address the audit concern. `WCAG 1.4.4` doesn’t actually define a minimum font size - the main requirement is that text can scale to 200% without loss of content or functionality. Referencing `1.4.4` here is more by convention. 
TODO: Update this part of this issue to define the agreed minimum font-size to use.

## How to reproduce / Current behavior
1. Log in to OJS/OMP or OPS Editorial View.
2. Inspect any of these page (dashboard, any workflow tables, add reviewer, add discussion) in dev tools.
3. Body / table text shows `font-size: 0.75rem` or `.6875rem` or anything below `0.875rem (14px seems to be the acceptable font-size for webpages. Needs a decision on what should be the minimum size to use)`.

## What application are you using?
 OJS, OMP or OPS version 3.5

## Stack used 
- Device: Desktop
- OS: Windows/macOS
- Browser: Chrome browser/Firefox

## Testing method
- Method: Manual test

## Action
1. Review and update the font styles configured in ui-library.
2. Set the backend body text CSS variables depending on the agreed font-size to use for templates/forms.

## Additional information
<img width="1891" height="851" alt="Image" src="https://github.com/user-attachments/assets/a49337d7-9792-46fb-8d80-92e45ff15e5a" />

---

<img width="1612" height="824" alt="Image" src="https://github.com/user-attachments/assets/1a351031-8209-4d34-93e3-54b07299e625" />

---

<img width="1618" height="788" alt="Image" src="https://github.com/user-attachments/assets/2149219a-7824-40c7-b737-b33b91fcbf19" />

### WCAG Success Criteria
1.4.4 Resize Text (AA)
